extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authorization/endpoint/BasicAuthorizerResource.java

```diff
@@ -610,6 +612,24 @@ public Response getCachedSerializedGroupMap(
     return resourceHandler.getCachedGroupMappingMaps(authorizerName);
   }
 
+  @GET
+  @Path("/db/{authorizerName}/history")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ResourceFilters(BasicSecurityResourceFilter.class)
+  public Response getAuthorizerUpdateHistory(
```

Comment:
It looks like the only caller for this resource is the embedded test. 
Do we really need this API ?

Reply:
If you really feel strongly about having this API, I guess a revert PR followed by a new patch for adding an API makes more sense no?